db/migrate/20161011074804_add_current_organization_to_user.rb

```diff
@@ -0,0 +1,14 @@
+class AddCurrentOrganizationToUser < ActiveRecord::Migration
```

Comment:
You're missing Frozen?
http://img.lum.dolimg.com/v1/images/homepage_hero_frozen_winter_18c81bd7.jpeg?region=0%2C0%2C480%2C240